### ISSUE TITLE
SentrySDK crash in captureMessage

### ISSUE THREAD

**Issue by eaigner** [getsentry/sentry-cocoa#603]
_Platform:_
- [ ]  iOS
- [ ]  tvOS
- [x]  MacOS
- [ ]  watchOS

_sentry-cocoa installed with:_
- [ ] CocoaPods
- [ ] Carthage
- [x] Manually

Version of sentry-cocoa:
**5.1.6**

---
I have following issue:

Here is the partial stacktrace for the crash

~~~
OS Version: macOS 10.15.5 (19F101)
Report Version: 104

Exception Type: EXC_BAD_ACCESS (SIGBUS)
Exception Codes: BUS_NOOP at 0x00000001092759d0
Crashed Thread: 10

Application Specific Information:
Exception 1, Code 0, Subcode 8 >
Attempted to dereference garbage pointer 0x1092759d0.

Thread 10 Crashed:
0   libobjc.A.dylib                 0xfffeb66d1829      objc_msgSend
1   Sentry                          0x10927373b         -[SentryClient sendEvent:withScope:alwaysAttachStacktrace:]
2   Sentry                          0x10927339a         -[SentryClient captureMessage:withScope:]
3   Sentry                          0x109283c7c         -[SentryHub captureMessage:withScope:]
4   Sentry                          0x1092954fc         +[SentrySDK captureMessage:withScope:]
5   Sentry                          0x109295353         +[SentrySDK captureMessage:]
~~~

**Comment by eaigner** [comment]
Probably can't rule out a garbage value coming from the outside though.

**Comment by philipphofmann** [comment]
Thanks, @eaigner for reporting this issue. Do you have more context around the issue? When does it happen?

**Comment by eaigner** [comment]
No thats all I have. Happened on another machine.

**Comment by philipphofmann** [comment]
Did it only happen once or multiple times @eaigner ?

**Comment by eaigner** [comment]
Once since I upgraded to 5.1.6

**Comment by eaigner** [comment]
Correction. Got 3 more since upgrade to 5.1.6.

**Comment by eaigner** [comment]
I found this PR #583 that changed something in the last 14 days relevant to this stacktrace. Maybe the issue is in there somewhere?

**Comment by philipphofmann** [comment]
We solved one issue of #583 with #605, but this should not be related to the crash you mention in here.

**Comment by eaigner** [comment]
I think I'll revert to 5.1.0 for now, this happens way to often. An interesting thing is that it only seems to happen in the MAS version of our app.
